### DISCUSSION TITLE
清爽首页 在 Safari 浏览器下 transform 的时候圆角变方框

### DISCUSSION THREAD

**Discussion by Agoines** [the1812/Bilibili-Evolved#3528] [综合讨论 / General]
在 Safari 下，transform 的时候圆角变方框，经过测试添加了一行 `-webkit-transform:rotate(0deg);` 可以解决这个问题，但是不知道对应的清爽首页源码在哪里

**Comment by the1812** [comment]
https://github.com/the1812/Bilibili-Evolved/tree/preview-fixes/registry/lib/components/style/home-redesign/fresh

**Reply by Agoines** [reply]
明白了，已经提交了一个修复这个小问题的 pr
